docs/tutorials/aws.md

```diff
@@ -84,7 +84,7 @@ spec:
         args:
         - --source=service
         - --source=ingress
-        - --domain-filter=external-dns-test.my-org.com. # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
+        - --domain-filter=external-dns-test.my-org.com # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
```

Comment:
I would not tutorials before the new release, the tutorial is anyway tied to the previous release